### PR TITLE
TF-1194 Fix create first default identity make a null identity

### PR DESCRIPTION
**Pull request by sherlockvn**
Before: 

https://user-images.githubusercontent.com/43041967/213383789-6d9fdada-f65a-4082-ae70-bfb41e5cd29e.mp4

After:

https://user-images.githubusercontent.com/43041967/213383923-9259613c-5647-4a3c-8a01-7fde691d2f70.mp4

### CODE REVIEW

**Review by nqhhdev — APPROVED** [review]

File: lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart [hunk 1]
```diff
@@ -37,8 +37,9 @@ class CreateNewDefaultIdentityInteractor {
     final listIdentities = await _identityRepository
         .getAllIdentities(
             accountId, 
-            properties: Properties({'sortOrder'})
+            properties: Properties({'sortOrder', 'mayDelete'})
```
[the remark this answers is not in the document]
Comment (hoangdat) [reply lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart:40]: image

File: lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart [hunk 2]
```diff
@@ -37,8 +37,9 @@ class CreateNewDefaultIdentityInteractor {
...
         );
+    listIdentities.identities?.removeWhere((identity) => identity.mayDelete != true);
```
Comment (hoangdat) [inline lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart:42]: should wrap `(identity) => identity.mayDelete != true` in a method for more understand

File: lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart [hunk 3]
```diff
@@ -39,10 +39,14 @@ class CreateNewDefaultIdentityInteractor {
             accountId, 
             properties: Properties({'sortOrder', 'mayDelete'})
         );
-    listIdentities.identities?.removeWhere((identity) => identity.mayDelete != true);
+    listIdentities.identities?.removeWhere((identity) => _isIdentityUnDeletable(identity));
```
Comment (hoangdat) [inline lib/features/manage_account/domain/usecases/create_new_default_identity_interactor.dart:42]:
```suggestion
listIdentities.identities?.removeWhere(_isIdentityUnDeletable);
```

**Review by dab246 — APPROVED** [review]
lgtm

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**